webapp/css/style.css

```diff
@@ -1 +1,17 @@
-/* Enter your custom styles here */
\ No newline at end of file
+/* Enter your custom styles here */
+
+.listSizing {
+    width: 25rem !important;
+    overflow: scroll;
+    height: 30rem;
+}
+
+.globalMargin {
+    margin: 0rem 6rem;
+}
+
+.headerList {
+    width: 9rem !important;
+    font-size: 1.5rem;
+    padding: 2%;
```

Comment:
Let's be consistent and use only one metrics for styling. In this case use or rem or %.